package.json

```diff
@@ -42,7 +42,7 @@
     "sass-loader": "^5.0.1",
     "vue-loader": "^11.1.4",
     "vue-template-compiler": "^2.2.1",
-    "vuetify": "^0.11.1",
+    "vuetify": "^0.12.0",
     "webpack": "^2.2.0",
```

Comment:
Nie wiem, czy dobrze rozumiem, ale masz to wrzucone tutaj i inline'owo w index.html - to jest coś innego, czy się powtarzasz?